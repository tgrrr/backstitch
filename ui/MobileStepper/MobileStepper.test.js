import React from 'react'
import renderer from 'react-test-renderer'
import { MobileStepper } from '.'

it('MobileStepper: default', () => {
    const component = renderer.create(
        <MobileStepper
            steps={5}
        >
            <>
                MobileStepper
            </>
        </MobileStepper>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
