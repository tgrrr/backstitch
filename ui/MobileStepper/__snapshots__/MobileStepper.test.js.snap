// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MobileStepper: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Mobilestepper "
    data-testid="Mobilestepper"
  >
    <div
      className="MuiPaper-root MuiMobileStepper-root MuiMobileStepper-positionBottom MuiPaper-elevation0"
    >
      <div
        className="MuiMobileStepper-dots"
      >
        <div
          className="MuiMobileStepper-dot MuiMobileStepper-dotActive"
        />
      </div>
    </div>
  </div>
</div>
`;
